sky/provision/kubernetes/utils.py

```diff
@@ -601,9 +603,12 @@ def check_credentials(timeout: int = kubernetes.API_TIMEOUT) -> \
         bool: True if credentials are valid, False otherwise
         str: Error message if credentials are invalid, None otherwise
     """
+
     try:
         ns = get_current_kube_config_context_namespace()
-        kubernetes.core_api().list_namespaced_pod(ns, _request_timeout=timeout)
+        with provision_logging.set_logging_level('urllib3', logging.ERROR):
```

Comment:
We may also need to add this to other first-touch points:
e.g., `sky launch -c test --cloud kubernetes` hits `get_kubernetes_nodes()`, which still gives the warning:

```
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x135c5a670>: Failed to establish a new connection: [Errno 61] Connection refused')': /api/v1/nodes
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x135c5a760>: Failed to establish a new connection: [Errno 61] Connection refused')': /api/v1/nodes
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x135c76550>: Failed to establish a new connection: [Errno 61] Connection refused')': /api/v1/nodes
Traceback (most recent call last):
....
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/provision/kubernetes/utils.py", line 401, in check_instance_fits
    nodes = get_kubernetes_nodes()
  File "/Users/romilb/Romil/Berkeley/Research/sky-experiments/sky/provision/kubernetes/utils.py", line 342, in get_kubernetes_nodes
    raise exceptions.ResourcesUnavailableError(
sky.exceptions.ResourcesUnavailableError: Timed out when trying to get node info from Kubernetes cluster. Please check if the cluster is healthy and retry.
```

Can you also check for `sky exec`, `sky jobs`, `sky serve` and other CLIs?

Reply:
edited so that all APIs from `kubernetes` are wrapped/decorated with `set_logging_level('urllib3`, logging.ERROR)`